flask_monitoringdashboard/core/cache.py

```diff
@@ -64,29 +65,60 @@ def init_cache():
             )
 
 
+def add_to_cache(endpoint_name):
+    """
+        This should be added to all endpoint no directly hosted in Flask before the first request.
```

Comment:
what are the endpoints not directly hosted in Flask? I don't understand.

Reply:
My apologies for the lack of clarity. In my use case, I am using flask to proxy a few requests to a different server which I also want to monitor within FlaskMonitoring dashboard. Hence, I want to be able to populate the cache on will with a proper function